mesh/settings.py

```diff
@@ -42,6 +42,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'rest_framework',
+    'djoser',
```

Comment:
These should be added to the pipfile if it's required for the project, right?

Reply:
Correct, just fixed that. Thank you!